packages/tools/devtools/devtools-view/src/components/TelemetryView.tsx

```diff
@@ -520,6 +521,7 @@ function FilteredTelemetryView(props: FilteredTelemetryViewProps): React.ReactEl
 		<>
 			<div style={{ display: "flex", gap: "10px" }}>
 				<Dropdown
+					aria-label="Filter Category"
```

Comment:
For consistency with "Event Name Filter" below.
```suggestion
					aria-label="Category Filter"
```